client/src/modules/stock/StockEntryExitType.service.js

```diff
@@ -0,0 +1,100 @@
+angular.module('bhima.services')
+.service('StockEntryExitTypeService', StockEntryExitTypeService);
+
+StockEntryExitTypeService.$inject = [];
+
+function StockEntryExitTypeService() {
+  var $ctrl = this;
```

Comment:
Since this is a service, could you call this `service` instead of `$ctrl`?  It will confuse future developers for sure.

Reply:
You never updated this!